common/transformations/camera.py

```diff
@@ -1,29 +1,67 @@
 import numpy as np
+
 import common.transformations.orientation as orient
+from common.hardware import TICI
 
-FULL_FRAME_SIZE = (1164, 874)
-W, H = FULL_FRAME_SIZE[0], FULL_FRAME_SIZE[1]
-eon_focal_length = FOCAL = 910.0
+## -- hardcoded hardware params --
+eon_f_focal_length = 910.0
+eon_d_focal_length = 860.0
+leon_d_focal_length = 650.0
+tici_f_focal_length = 2648.0
+tici_e_focal_length = tici_d_focal_length = 567.0 # probably wrong? magnification is not consistent across frame
 
-# aka 'K' aka camera_frame_from_view_frame
-eon_intrinsics = np.array([
-  [FOCAL,   0.,   W/2.],
-  [  0.,  FOCAL,  H/2.],
-  [  0.,    0.,     1.]])
+eon_f_frame_size = (1164, 874)
+eon_d_frame_size = (1152, 864)
+leon_d_frame_size = (816, 612)
+tici_f_frame_size = tici_e_frame_size = tici_d_frame_size = (1928, 1208)
 
+# aka 'K' aka camera_frame_from_view_frame
+eon_fcam_intrinsics = np.array([
+  [eon_f_focal_length,  0.0,  float(eon_f_frame_size[0])/2],
+  [0.0,  eon_f_focal_length,  float(eon_f_frame_size[1])/2],
+  [0.0,  0.0,                                          1.0]])
+eon_intrinsics = eon_fcam_intrinsics # xx
 
 leon_dcam_intrinsics = np.array([
-  [650,   0,   816//2],
-  [  0,  650,  612//2],
-  [  0,    0,     1]])
+  [leon_d_focal_length,  0.0,  float(leon_d_frame_size[0])/2],
+  [0.0,  leon_d_focal_length,  float(leon_d_frame_size[1])/2],
```

Comment:
You changed this array from int to float. If that was unintentional make sure float is what we want. I see the eon_intrinsics are float too.

Reply:
I think the camera intrinsics are mostly used with other float quantities such as calib.
And yes the eon_intrinsics has been float.